sdk/test-utils/recorder-new/src/core-v2-recorder.ts

```diff
@@ -171,7 +193,7 @@ export class TestProxyHttpClient {
   /**
    * Call this method to ping the proxy-tool with a stop request, this helps saving the recording in record mode.
    */
-  async stop(): Promise<void> {
+  async stop(variables?: Record<string, string>): Promise<void> {
```

Comment:
What's the purpose of the variables parameter here when it is a stateful property of the class?

Reply:
I was thinking of giving more control to the users by allowing them to override by passing all the variables at once.
But I guess this adds more confusion than convenience.

Will remove it.